src/senaite/core/tests/doctests/API_mail.rst

```diff
@@ -232,3 +232,43 @@ This function composes a new MIME message:
     5/sfV5M/kISv300AAAAASUVORK5CYII=
     ...
     <BLANKLINE>
+
+By default, the body is not encoded as html:
```

Comment:
Typo:
"By default, the body is not encoded as *text*"

Reply:
Apologies, did not read correctly 🙈  I have issues reading these `not` statements...

Reply:
- [ ] Enable this checkbox to disable `not` statements 😅